HomeWorks/java/lesson8/Task1HW8.java

```diff
@@ -0,0 +1,10 @@
+package lesson8;
+
+public class Task1HW8 {
+
+    //Top 4 most effective locators for web elements on GitHub's Code tab
+  String codeTab = "WebElement element = driver.findElement (By.id ("code-tab"))";
+  String goToFileButton = "WebElement element = driver.findElement (By.linkText ("Go to file"))";
+  String buttonNavigationMenu = "WebElement element = driver.findElement (By.id ("dialog-show-dialog-d92d47ee-e093-489f-99d5-cb8f6f2f5e82"))";
```

Comment:
by id - is the fastest locator but here id value is not informative and there is suspicion that this id can be changeable.